gradle.properties

```
@@ -26,26 +26,26 @@ release_type=release
 # https://maven.parchmentmc.org/org/parchmentmc/data
 # Forge dependencies use maven's version range spec:
 # https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html
-minecraft_version=1.21.7
-parchment_version=1.21.6-2025.06.29
+minecraft_version=1.21.8
+parchment_version=1.21.8-2025.07.20
 supported_mc_versions=
 
 fabric_loader_version=0.16.14
-fabric_api_version=0.128.2+1.21.7
+fabric_api_version=0.130.0+1.21.8
 fabric_loader_req=>=0.14.0
-fabric_mc_req=>1.21.7-
+fabric_mc_req=>1.21.8-
 fabric_supported_mc_versions=
 
-neoforge_version=21.7.10-beta
+neoforge_version=21.8.25
 neoforge_pr=
-neoforge_mc_req=[1.21.7,)
+neoforge_mc_req=[1.21.8,)
 neoforge_loader_req=[1,)
-neoforge_req=[21.7.3-beta,)
+neoforge_req=[21.8.25,)
 neoforge_supported_mc_versions=
 
 # Other dependencies
 # https://maven.terraformersmc.com/releases/com/terraformersmc/modmenu
 # https://mvnrepository.com/artifact/me.shedaniel.cloth/cloth-config?repo=architectury
 # FIXME
 modmenu_version=15.0.0-beta.3
```

This can be bumped now too, although it isn't strictly tied to 1.20.8 and would also be a valid bump on 1.20.6.

```suggestion
modmenu_version=15.0.0
```

Since we don't bundle modmenu in our jar, and this specific version is only used locally in the dev environment, it doesn't matter too much.